servers/audio/audio_stream.cpp

```diff
@@ -83,6 +83,11 @@ int AudioStreamPlayback::mix(AudioFrame *p_buffer, float p_rate_scale, int p_fra
 	return 0;
 }
 
+void AudioStreamPlayback::tag_used_streams() {
+	if (GDVIRTUAL_CALL(_tag_used_streams)) {
+	}
```

Comment:
Why is this a condition if it does nothing.
(although maybe it silenced some warning, in which case nevermind...)